multi-agent/tests/e2e/test_session_stress.py

```
@@ -296,17 +397,52 @@ def delete_session(self, session_id: str) -> bool:
 # SAMPLE BLUEPRINT
 # =============================================================================
 
-def get_stress_test_blueprint() -> Dict:
+def _normalize_llm_ref(llm_ref: str) -> str:
+    """Ensure catalog LLM refs use the $ref: prefix Temporal mini-blueprints require."""
+    ref = (llm_ref or "").strip()
+    if not ref:
+        raise ValueError("llm_ref must not be empty")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)